esiclient/tests/unit/v1/test_node_network.py

```diff
@@ -469,6 +473,24 @@ def test_take_action_no_port_or_network_or_trunk_exception(self):
             'ERROR: You must specify either network, port, or trunk',
             self.cmd.take_action, parsed_args)
 
+    def test_take_action_invalid_network_exception(self, mock_gfnifp):
+        self.app.client_manager.baremetal.node.get.\
+            return_value = self.node
+        self.app.client_manager.baremetal.port.list.\
+            return_value = [self.port1]
+
+        arglist = ['node1', '--network', 'test_network_invalid']
+        verifylist = []
+
+        parsed_args = self.check_parser(self.cmd, arglist, verifylist)
+#       results = self.cmd.take_action(parsed_args)
+#       print("Invalid network exception results {}".format(results))
```

Comment:
Make sure to remove these commented out lines!

Reply:
Sure! They are just for my test purpose.